### ISSUE TITLE
Inability to walk on heels

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6792]
**Preferred term label:**
Inability to walk on heels

**Synonyms**
Inability to heel walk

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://omim.org/entry/617114

**Comment by pnrobinson** [comment]
I think this is a synonym of https://hpo.jax.org/app/browse/term/HP:0009027, adding it there.
